android/app/src/main/AndroidManifest.xml

```diff
@@ -436,7 +436,6 @@
         android:name="app.organicmaps.MapPlaceholderActivity"/>
     <service
         android:name="app.organicmaps.car.CarAppService"
-        android:foregroundServiceType="location"
```

Comment:
Is it a correct fix? Will AA work properly when the app is in the background on the device (or the device is locked)? Someone from Google describes a different fix here: https://issuetracker.google.com/issues/322622698#comment29

Reply:
This is exactly what I've done. The rest part is already implemented on our side.